lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -5,6 +5,11 @@ module Cop
     module RSpec
       # Checks that `context` docstring starts with an allowed prefix.
       #
+      # The default list of prefixes is minimal. Users are encouraged to tailor
+      # the configuration to meet project needs. Other acceptable prefixes may
+      # include `if`, `unless`, `for`, `before`, `after`, or `during`.
+      #
+      # @see https://rspec.rubystyle.guide/#context-descriptions
       # @see https://github.com/reachlocal/rspec-style-guide#context-descriptions
```

Comment:
If you're up for making another correction, the namespace has changed from `reachlocal` to `rubocop-hq`.

Reply:
If it's the same content as https://rspec.rubystyle.guide, maybe it's better to have only one?

Reply:
Fair enough.

Reply:
I removed the github.com link, since the rubystyle.guide link has the same content in a better form. Changes pushed.